Sources/MountebankSwift/Models/Recreatable.swift

```diff
@@ -112,7 +113,24 @@ extension Recreatable {
 
 extension String: Recreatable {
     var recreatable: String {
-        debugDescription
+        contains(where: \.isNewline)
+            ? multilineRecreatable
+            : debugDescription
+    }
+
+    private var multilineRecreatable: String {
+        let description = description
+            .split(separator: "\n")
+            .map {
+                String(repeating: whitespace, count: indent) + $0
+            }
+            .joined(separator: "\n")
+
+        return """
+        \"\"\"
+        \(description)
+        \(String(repeating: whitespace, count: indent))\"\"\"
+        """
```

Comment:
This improves multiline strings a lot. 

"some\nmultiline\nstring"

not becomes
"""
some
multiline
string
"""

This is used for injection